DOCKER_MORE_CLOSELY.md

```diff
@@ -0,0 +1,44 @@
+# Docker More Closely
+
+## Apple M1 Platform
+
+***Attention:** For using Docker commands in Apple M1 environments!*
+
+### Enviroment Variable For Apple M1 Platform
+
+To set the Docker platform environment variable in your terminal tab, run:
+
+```bash
+# set env variable for your shell
+$ export DOCKER_DEFAULT_PLATFORM=linux/amd64
+```
+
+### Docker Compose Override File For Apple M1 Platform
+
+For Docker compose `up` or `build` commands, you can use our Apple M1 override file that specifies the M1 platform:
+
+```bash
+# in main folder
+
+# for development
+$ docker compose -f docker-compose.yml -f docker-compose.override.yml -f docker-compose.apple-m1.override.yml up
+
+# for production
+$ docker compose -f docker-compose.yml -f docker-compose.apple-m1.override.yml up
```

Comment:
```suggestion
$ docker-compose -f docker-compose.yml -f docker-compose.apple-m1.override.yml up
```

Reply:
The new command is without the dash @Mogge , but both will work for a while …
See https://docs.docker.com/compose/cli-command-compatibility/